ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/VariablesList.java

```diff
@@ -80,7 +80,7 @@ public String toString() {
   @JsonCreator
   public VariablesList(
       @JsonProperty("statementText") final String statementText,
-      @JsonProperty("properties") final List<Variable> variables
+      @JsonProperty("variables") final List<Variable> variables
```

Comment:
this has never been released, right? should be safe to change

Reply:
Yeap, never released, so we're good here.